samples/wifi/sta/boards/thingy91x_nrf9151_ns.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_WIFI_READY_LIB=n
```

Comment:
any special reason for disabling this library? it's handy to track Wi-Fi usability.